platform/src/main/java/org/hillview/storage/GenericLogs.java

```diff
@@ -95,12 +108,30 @@ public ITable load() {
                 String[] fields = new String[this.columns.length];
                 while (true) {
                     String line = reader.readLine();
-                    if (line == null)
+                    if (line == null) {
+                        this.parse(this.genLog.multiLine, fields);
+                        this.append(fields);
                         break;
+                    }
                     if (line.trim().isEmpty())
                         continue;
-                    this.parse(line, fields);
-                    this.append(fields);
+                    Matcher matchDate = datePattern.matcher(line);
+                    if (matchDate.find()) {
```

Comment:
So the heuristic is that if there is no date this is a continuation of the previous line?

Reply:
@mbudiu-vmw Yes.

Reply:
Hopefully it won't be too difficult to merge this with the other PR I have submitted.

Reply:
Sure.